octoprint_mrbeam/templates/wizard/firstrun_start.jinja2

```diff
@@ -1,4 +1,4 @@
-<div data-test="first-run-wizard">
+<div class="row" data-test="first-run-wizard">
```

Comment:
Why you added the row here? This will affect the whole wizard

Reply:
This will affect just the first wizard window and it is for rightful styling format so that all the wizard pages start from the same level (It has to do with the paddings added by bootstrap)